packages/shared/styles/components/atoms/SfInput.scss

```diff
@@ -101,7 +101,7 @@
       var(--font-family--secondary)
     );
     text-align: var(--input-text-align);
-    &:focus, &:placeholder-shown {
+    &:focus, &:placeholder-shown, &:-webkit-autofill, &:-webkit-autofill:hover, &:-webkit-autofill:focus {
```

Comment:
why not use `focus-within`?

Reply:
I didn't know that it could work in this case. Does it mean that it should detect focus on the text node inside the input element?

Reply:
@mayashavin  I tried using focus-within, but it diddn't work, with autofill it's okay.